build_and_install_images.sh

```diff
@@ -27,6 +27,7 @@ do
 	cmd="easeml create module --type $t --source upload --source-address $d --id opt-$d --label opt-$d --name opt-$d --config $config_file" 
 	echo $cmd
 	eval $cmd
+	#read -p "Press enter to continue"
```

Comment:
Temp comments like this should be deleted.

Reply:
fixed